packages/controllers/src/snaps/SnapController.ts

```diff
@@ -16,6 +16,7 @@ import {
   ValidPermission,
 } from '@metamask/controllers';
 import { ErrorJSON, SnapData, SnapId } from '@metamask/snap-types';
+import { HandlerType } from '@metamask/execution-environments';
```

Comment:
We're really including whole execution environments package with all of its dependencies for 2 enums.
Could you please extract both of them to snap-utils?